src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Microsoft.VisualStudio.Web.CodeGeneration.Tools.csproj

```diff
@@ -29,7 +29,7 @@
     <PackageReference Include="Microsoft.Extensions.CommandLineUtils" Version="$(CommandLineUtilsLegacyVersion)" />
     <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="$(PlatformAbstractionsLegacyVersion)" />
     <PackageReference Include="Microsoft.Extensions.FileProviders.Physical" Version="$(FileProvidersLegacyVersion)" />
-    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonSoftVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonSoftLegacyVersion)" />
```

Comment:
The `Microsoft.VisualStudio.Web.CodeGeneration.Tools` package needs to target `netcoreapp1.0` as its a CLI tool. 
`Newtonsoft.Json 10.0.1` takes dependencies on `System.*` packges with version `4.3.0` which is part of `1.1.x` train